Restructure the codebase stewardship activity (#1129)
publiccodenet/about: activities/codebase-stewardship/index.md

# Codebase stewardship

> Codebase stewardship is what we call the way we interact with the public code codebase communities that aims to meet the Standard for Public Code.

Reviewer: ```suggestion
Codebase stewardship is what we call the way we support public code codebase communities as they aim and work to meet the Standard for Public Code.
```

Reviewer: maybe even just `as they work to meet the standard`?

> Our approach of being stewards is highly intentional.

Reviewer: This sentence is a bit unclear to me

Author: Even in the context of the full paragraph? I was going for to make a statement, and then explaining it in the following sentences.

> Our approach of being stewards is highly intentional.

Reviewer: ```suggestion
Our approach of relating to the codebase community from the position of being stewards is highly intentional.
```

We have no ambition in claiming ownership of or running a codebase, but rather our goal is to help the community be successful in what they want to achieve.
Thus, our approach is more similar to a coach, mentor or advisor rather than a hands-on developer or project manager.

> Sometimes it also means that we will act as a neutral party that the community can escalate issues to, or to arbitrate on.

Reviewer: ```suggestion
Sometimes we act as a neutral party that can arbitrate on issues that the community escalates to us.
```
